src/utils/prettyNum.tests.ts

```diff
@@ -0,0 +1,14 @@
+import o from 'ospec';
+import { prettyNum } from './prettyNum';
+
+o.spec('regEscape()', () => {
+	o('Escapes strings', () => {
+		o(prettyNum()).equals('0');
+		o(prettyNum(42)).equals('42');
+		o(prettyNum(123456789)).equals('123.456.789');
+		o(prettyNum(123456.789, 2, '.', ',')).equals('123.456,79');
+		o(prettyNum(1234.56789, 3, ',', '.')).equals('1,234.568');
+		o(prettyNum(123456.789, 2, '', ',')).equals('123456,79');
+		o(prettyNum(123456.789, 2, '', '')).equals('12345679');
+	});
```

Comment:
BTW, það vantar alveg að testa tölur sem eru minni en 1
```suggestion
		o(prettyNum(-1234.56)).equals('-1,234.57');
		o(prettyNum(-0.12)).equals('-0.12');
	});
```

...etc. með fyrirvara um default höndlun á tugabrotum.

Reply:
komin test fyrir mínus tölur